src-cpp/CMakeLists.txt

```diff
@@ -51,7 +48,7 @@ set(PKG_CONFIG_CFLAGS
     "-I\${includedir}"
 )
 set(PKG_CONFIG_LIBS
-    "-L\${libdir} -lrdkafka++"
```

Comment:
Don't remove `-lrdkafka++` here. The pkg-config file shall provide a valid link list for users of the (shared) library. Omitting  this parameter would lead to linking errors.

Reply:
done